data/scenarios/Tutorials/stock.yaml

```diff
@@ -1,18 +1,18 @@
 version: 1
-name: Require inventory
+name: Stock
 description: |
-  Learn how to require inventory when building robots.
+  Learn how to stock inventory when building robots.
 objectives:
   - goal:
       - |
         In the previous tutorial challenge, you learned how to use `require`{=snippet}
         to require specific devices to be equipped.
-        Sometimes, instead of requiring equipped devices, you require supplies in your inventory.
+        Sometimes, instead of requiring equipped devices, you stock supplies in your inventory.
```

Comment:
Happy to try and reword this if needed. It used to follow nicely that you just learnt about `requiring` devices and now you are `requiring` inventory but it now doesn't follow as well.

Reply:
The connection still makes sense since both are about providing entities to a built robot.  How about something like "Sometimes, instead of requiring entities to be equipped as devices, you need them to be stocked in the inventory"?